proteus/Domain.py

```diff
@@ -514,7 +605,7 @@ class PlanarStraightLineGraphDomain(D_base):
     2D domains described by planar straight line graphs.
     """
 
-    def __init__(self, fileprefix=None, vertices=None, segments=None, holes=None, regions=None, vertexFlags=None,
+    def __init__(self, fileprefix=None, vertices=None, segments=None, facets=None, holes=None, regions=None, vertexFlags=None,
```

Comment:
I added the facets object primarily because I wanted to have a general way to treat model boundaries if 3D mesh-to-model classification was implemented. I'll remove it for now.